src/Microsoft.DotNet.Build.Tasks/PackageFiles/crossgen.sh

```diff
@@ -54,13 +54,17 @@ crossgen_single()
 {
     __file=$1
     if [[ $__file != *.ni.dll && $__file != *.ni.exe ]]; then
-        $__crossgen /Platform_Assemblies_Paths $__sharedFxDir:$__toolsDir /nologo /MissingDependenciesOK /ReadyToRun $__file > /dev/null
-        if [ $? -eq 0 ]; then
-            __outname="${__file/.dll/.ni.dll}"
-            __outname="${__outname/.exe/.ni.exe}"
-            echo "$__file -> $__outname"
+        if [[ ($__file == *.dll && -e ${__file/.dll/.ni.dll}) || ($__file == *.exe && -e ${__file/.exe/.ni.exe}) ]]; then
```

Comment:
It helped locally with the latest buildtools.  Do you know when that change went in?